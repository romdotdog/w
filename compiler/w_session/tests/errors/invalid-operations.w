fn main(a: f32, b: f32, c: f64, d: f64) {
    a % b;
    a & b;
    a | b;
    a ^ b;
    a >> b;
    a << b;
    
    c % d;
    c & d;
    c | d;
    c ^ d;
    c >> d;
    c << d;
}