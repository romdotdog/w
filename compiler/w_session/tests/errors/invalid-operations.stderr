tests/errors/invalid-operations.w:2:10: error: invalid operation
    a % b;
         ^
tests/errors/invalid-operations.w:3:10: error: invalid operation
    a & b;
         ^
tests/errors/invalid-operations.w:4:10: error: invalid operation
    a | b;
         ^
tests/errors/invalid-operations.w:5:10: error: invalid operation
    a ^ b;
         ^
tests/errors/invalid-operations.w:6:11: error: invalid operation
    a >> b;
          ^
tests/errors/invalid-operations.w:7:11: error: invalid operation
    a << b;
          ^
tests/errors/invalid-operations.w:9:10: error: invalid operation
    c % d;
         ^
tests/errors/invalid-operations.w:10:10: error: invalid operation
    c & d;
         ^
tests/errors/invalid-operations.w:11:10: error: invalid operation
    c | d;
         ^
tests/errors/invalid-operations.w:12:10: error: invalid operation
    c ^ d;
         ^
tests/errors/invalid-operations.w:13:11: error: invalid operation
    c >> d;
          ^
tests/errors/invalid-operations.w:14:11: error: invalid operation
    c << d;
          ^
